components/scg/src/scg-scene.js

```diff
@@ -637,7 +637,7 @@ SCg.Scene.prototype = {
 
     isSelectedObjectAllArcsOrAllNodes: function () {
```

Comment:
if we want to get rid of 'arc' usages in places that suppose to mean connector then you can use `(?<!se)arc(?!ommon)` to search arc usages

Reply:
Fixed